Tests/SwiftDocCUtilitiesTests/ConvertActionTests.swift

```diff
@@ -1981,7 +1981,7 @@ class ConvertActionTests: XCTestCase {
         let indexFromConvertAction = try NavigatorIndex.readNavigatorIndex(url: indexURL)
         XCTAssertEqual(indexFromConvertAction.count, 37)
         
-        try FileManager.default.removeItem(at: indexURL)
+        try indexFromConvertAction.clearIndex()
```

Comment:
Since we do not need the logic in normal env and we already have `@testable import`, I think we can directly call it and remove `clearIndex ` defined on `NavigatorIndex.swift`.

```suggestion
        indexFromConvertAction.environment?.close()
        try FileManager.default.removeItem(at: self.url)
```